test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/returns/CheckYourReturnControllerSpec.scala

```diff
@@ -34,19 +37,36 @@ import uk.gov.hmrc.plasticpackagingtax.returns.models.response.FlashKeys
 import uk.gov.hmrc.plasticpackagingtax.returns.views.html.returns.check_your_return_page
 import uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents
 
+import scala.concurrent.Future
+
 class CheckYourReturnControllerSpec extends ControllerSpec {
 
   private val mcc  = stubMessagesControllerComponents()
   private val page = mock[check_your_return_page]
 
+  private val resettableMockMetrics    = mock[Metrics]
+  private val mockDefaultRegistry      = mock[MetricRegistry]
+  private val submissionSuccessCounter = mock[Counter]
+  private val submissionFailedCounter  = mock[Counter]
+
+  {
+    when(resettableMockMetrics.defaultRegistry).thenReturn(mockDefaultRegistry)
```

Comment:
Metrics fought back alot. Made them reset able to workout what was been asserted.